docker-gc

```diff
@@ -283,3 +306,40 @@ else
     xargs -n 1 $DOCKER rmi $FORCE_IMAGE_FLAG < images.reap &>/dev/null || true
 fi
 
+if [[ $REMOVE_VOLUMES -gt 0 ]]; then
+    set +e
+    $DOCKER volume ls --filter "dangling=true" -q &> /dev/null
+    VOLUMES=$?
+    set -e
```

Comment:
Here we use set and unset ```e``` because ```docker volume``` was added in docker 1.9, so I wanted the program to stop and fail. (if I recall correctly this project used to use an older version at the time I have started with this feature). It might be not necessary now, as the image of docker-gc is currently build on a newer version....